codecparsers/bitReader.cpp

```diff
@@ -64,6 +64,13 @@ void BitReader::loadDataToCache(uint32_t nbytes)
     m_bitsInCache = nbytes << 3;
 }
 
+inline void BitReader::reload()
+{
+    uint32_t remainingBytes = m_size - m_loadBytes;
```

Comment:
would better change uint32_t to int32_t to improve this function's robustnes. Else if m_size < m_loadBytes, remainingBytes will still > 0.